src/Test/Perf/infra/fetch_build.csx

```diff
@@ -4,39 +4,130 @@
 #load "../util/tools_util.csx"
 
 using System;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
 using System.Xml;
 
 // TODO: Use actual command line argument parser so we can have help text, etc...
-var branch = Args.Count() == 2 ? Args[0] : "master";
+var sourceFolder = Args[0];
 var destinationFolder = Args.Count() == 2 ? Args[1] : @"C:\Roslyn\Binaries\Release";
```

Comment:
The old perf system did a nice thing where it made a directory for each drop
ie, C:\roslyn\20150304.1; C:\rosly\20150304.2, etc.